### PR TITLE
Fix responsiveness for button on claims document upload form

### PR DESCRIPTION
**Pull request by molindoz**
Closes: [Sunsets #125](https://github.com/department-of-veterans-affairs/sunsets-team/issues/125)
### After

screen shot 2016-10-26 at 12 49 33 pm

### CODE REVIEW

File: src/js/disability-benefits/components/AddFilesForm.jsx [hunk 1]
```diff
@@ -105,12 +105,18 @@ class AddFilesForm extends React.Component {
...
             <Element name={`documentScroll${index}`}/>
             <div className="document-title-size">
-              <div className="document-title-header">
-                <h4 className="title">{file.name}</h4>
+              <div className="document-title-text-container">
+                <div className="document-title-header">
+                  <h4 className="title">{file.name}</h4>
+                </div>
+                <div className="document-size-text">
+                  <p className="size">{displayFileSize(file.size)}</p>
```
Comment (jbalboni) [inline src/js/disability-benefits/components/AddFilesForm.jsx]: Do we need the `p` tag in here?

File: src/js/disability-benefits/components/AddFilesForm.jsx [hunk 2]
```diff
@@ -105,12 +105,18 @@ class AddFilesForm extends React.Component {
           <div key={index} className="document-item-container">
             <Element name={`documentScroll${index}`}/>
             <div className="document-title-size">
-              <div className="document-title-header">
-                <h4 className="title">{file.name}</h4>
+              <div className="document-title-text-container">
+                <div className="document-title-header">
+                  <h4 className="title">{file.name}</h4>
```
Comment (jbalboni) [inline src/js/disability-benefits/components/AddFilesForm.jsx]: What happens if the file name is long enough to overlap the button? Does it wrap and look ok?

**Review by jbalboni — APPROVED** [review]
LGTM, had a couple questions

File: src/js/disability-benefits/components/AddFilesForm.jsx (hunk 2 above)
Comment (cagedcrown) [inline src/js/disability-benefits/components/AddFilesForm.jsx]: Good call, I added a width and a word-break property so the text won't go outside of the div.
screen shot 2016-10-26 at 1 35 02 pm

File: src/js/disability-benefits/components/AddFilesForm.jsx (hunk 1 above)
Comment (cagedcrown) [inline src/js/disability-benefits/components/AddFilesForm.jsx]: I just checked and the text-size is exactly the same without the `p` tag, so I can remove this.

**Pull request merged**